src/components/Router/Router.tsx

```diff
@@ -5,5 +5,5 @@ import { routesConfig } from './routes';
 
 export const Router = () => {
   const routes: React.ReactElement | null = useRoutes(routesConfig);
```

Comment:
null can't be used here. (Missed in the previous PR)

Reply:
using generic is not provided by this hook

Reply:
Please use this doc https://reactnavigation.org/docs/typescript/#annotating-useroute

Reply:
Discussed on the daily